src/locales/en.json

```diff
@@ -152,7 +153,7 @@
 
   "nav": {
     "category": "Categories",
```

Comment:
I think 'category' should become 'services', and 'configured' > 'my_services' to keep translation consistent between en and fr

Reply:
You're right, I forgot this part (it's noticed on the trello card as well), thanks 👍